src/arch/cuda/hybridAPI/hapi_impl.cpp

```diff
@@ -861,16 +911,19 @@ static inline void hapiWorkRequestStartTime(hapiWorkRequest* wr) {
 static inline void profileWorkRequestEvent(hapiWorkRequest* wr,
                                            WorkRequestStage event) {
 #ifdef HAPI_INSTRUMENT_WRS
-  if (initializedInstrument()) {
+#if CMK_SMP || CMK_MULTICORE
```

Comment:
Doesn't multicore also set CMK_SMP?

Reply:
You're right, will change this in a separate patch.